descriptors/ModuleDescriptor-template.json

```diff
@@ -167,7 +167,7 @@
     "dockerPull": false,
     "dockerArgs": {
       "HostConfig": {
-        "Memory": 357913941,
+        "Memory": 3221225472,
```

Comment:
This is 3 GiB (I wish we could put a comment there, but pure JSON doesn't support comments for some insane reason)